### ISSUE TITLE
Unable to use super getters/setters when targeting ES5

### ISSUE THREAD

**Issue by nomaed** [Microsoft/TypeScript#15842]
**TypeScript Version:**  2.3.2

**Code**

A perfectly valid and working JS code (ES2015), when transpiled down to ES5, causes typescript to output an error "Only public and protected methods of the base class are accessible via the 'super' keyword" on `super.prop` access in child class `Bar`.

According to issue #11575 I understand that this is due to regular properties not being part of the prototype, but getters and setters are on the prototype in the transpiled code.

No errors are reported when targeting ES2015.

```ts
class Foo {
    private _prop;

    get prop() { return this._prop; }
    set prop(value) { this._prop = value; }
}

class Bar extends Foo {
    constructor() {
        super();
    }

    get prop() {
        return `parent + ${super.prop}`; // error here
    }

    setProp() {
        super.prop = 'foofoo';  // error here
    }

}

const bar = new Bar();
bar.setProp();
console.log(bar.prop);
```
TS Playground [link](https://www.typescriptlang.org/play/#src=class%20Foo%20%7B%0D%0A%20%20%20%20private%20_prop%3B%0D%0A%0D%0A%20%20%20%20get%20prop()%20%7B%20return%20this._prop%3B%20%7D%0D%0A%20%20%20%20set%20prop(value)%20%7B%20this._prop%20%3D%20value%3B%20%7D%0D%0A%7D%0D%0A%0D%0Aclass%20Bar%20extends%20Foo%20%7B%0D%0A%20%20%20%20constructor()%20%7B%0D%0A%20%20%20%20%20%20%20%20super()%3B%0D%0A%20%20%20%20%7D%0D%0A%0D%0A%20%20%20%20get%20prop()%20%7B%0D%0A%20%20%20%20%20%20%20%20return%20%60parent%20%2B%20%24%7Bsuper.prop%7D%60%3B%0D%0A%20%20%20%20%7D%0D%0A%0D%0A%20%20%20%20setProp()%20%7B%0D%0A%20%20%20%20%20%20%20%20super.prop%20%3D%20'foofoo'%3B%0D%0A%20%20%20%20%7D%0D%0A%0D%0A%7D%0D%0A%0D%0Aconst%20bar%20%3D%20new%20Bar()%3B%0D%0Abar.setProp()%3B%0D%0Aconsole.log(bar.prop)%3B)

**Expected behavior:**
This should be transpiled down correctly to ES5.
Just like it works using Babel as can be seen in the [repl here](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015&targets=&browsers=&builtIns=false&debug=false&code=class%20Foo%20%7B%0D%0A%20%20%20%20get%20prop()%20%7B%20return%20this._prop%3B%20%7D%0D%0A%20%20%20%20set%20prop(value)%20%7B%20this._prop%20%3D%20value%3B%20%7D%0D%0A%7D%0D%0A%0D%0Aclass%20Bar%20extends%20Foo%20%7B%0D%0A%20%20%20%20constructor()%20%7B%0D%0A%20%20%20%20%20%20%20%20super()%3B%0D%0A%20%20%20%20%7D%0D%0A%0D%0A%20%20%20%20get%20prop()%20%7B%0D%0A%20%20%20%20%20%20%20%20return%20%60parent%20%2B%20%24%7Bsuper.prop%7D%60%3B%0D%0A%20%20%20%20%7D%0D%0A%0D%0A%20%20%20%20setProp()%20%7B%0D%0A%20%20%20%20%20%20%20%20super.prop%20%3D%20'foofoo'%3B%0D%0A%20%20%20%20%7D%0D%0A%0D%0A%7D%0D%0A%0D%0Aconst%20bar%20%3D%20new%20Bar()%3B%0D%0Abar.setProp()%3B%0D%0Aconsole.log(bar.prop)%3B%0D%0A)

**Actual behavior:**
Error: Only public and protected methods of the base class are accessible via the 'super' keyword.

**Comment by RyanCavanaugh** [comment]
Duplicate #338 and many others; please search before logging new issues. Thanks!
